en/simulation/multi-vehicle-simulation.md

```diff
@@ -39,88 +38,72 @@ To test an example setup, follow the below steps,
-            <arg name="x" value="1"/>
-            <arg name="y" value="0"/>
-            <arg name="z" value="0"/>
-            <arg name="R" value="0"/>
-            <arg name="P" value="0"/>
-            <arg name="Y" value="0"/>
-            <arg name="vehicle" value="$(arg vehicle)"/>
-            <arg name="rcS" value="$(arg rcS2)"/>
-            <arg name="mavlink_udp_port" value="14562"/>
-            <arg name="ID" value="$(arg ID)"/>
-        </include>
-
-        <include file="$(find mavros)/launch/node.launch">
-            <arg name="pluginlists_yaml" value="$(arg pluginlists_yaml)" />
-            <arg name="config_yaml" value="$(arg config_yaml)" />
-
-            <arg name="fcu_url" value="$(arg fcu_url)" />
-            <arg name="gcs_url" value="$(arg gcs_url)" />
-            <arg name="tgt_system" value="$(arg tgt_system)" />
-            <arg name="tgt_component" value="$(arg tgt_component)" />
-        </include>
-    </group>
-    ```
-To simulate more vehicles, add more blocks with the appropriate changes in the startup files, and launch file's parameters.
\ No newline at end of file
+  * runs a mavros node
+  ```
+      <!-- MAVROS -->
+      <include file="$(find mavros)/launch/px4.launch">
+          <arg name="fcu_url" value="$(arg fcu_url)"/>
+          <arg name="gcs_url" value=""/>
+          <arg name="tgt_system" value="$(arg ID)"/>
+          <arg name="tgt_component" value="1"/>
+      </include>
+  ```
+  * Note: The complete block for each vehicle is enclosed in a set of `<group>` tags to seperate the ROS namespaces of the vehicles.
+
+To add a third iris to this simulation there are two main compenents to consider:
+* add `UAV3` to multi_uav_mavros_sitl.launch
+  * duplicate the group of either existing vehicle (`UAV1` or `UAV2`)
+  * increment the `ID` arg to `3`
+  * select a different port for `mavlink_udp_port` arg for communication with Gazebo
+  * selects ports for MAVROS communication by modifying both port numbers in the `fcu_url` arg
+* create a startup file, and change the file as follows:
+  * make a copy of an existing iris rcS startup file (`iris_1` or `iris_2`) and rename it `iris_3`
+  * `MAV_SYS_ID` value to `3`
+  * `SITL_UDP_PRT` value to match that of the `mavlink_udp_port` launch file arg
+  * the first `mavlink start` port and the `mavlink stream` port values to the same values, which is to be used for QGC communication
+  * the second `mavlink start` ports need to match those used in the launch file `fcu_url` arg (Note: Be aware of which port is src and dst for the different endpoints.)
+
+Additional resources:
+* See [Simulation](../simulation/README.md) for a discription of the UDP port configuraion.
+* See [URDF in Gazebo](http://gazebosim.org/tutorials/?tut=ros_urdf) for more information about spawning the model with xacro.
+* See [RotorS](https://github.com/ethz-asl/rotors_simulator/tree/master/rotors_description/urdf) for more xacro models.
```

Comment:
Any reason why the S in RotorS is capitalised

Reply:
Not sure why, but that's how I've seen it spelled. https://github.com/ethz-asl/rotors_simulator/blob/master/README.md

Reply:
OK, thanks